framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/HACepScenarioImpl.java

```diff
@@ -162,20 +189,52 @@ private File downloadAndUnzipAMQStreams() {
         final Unzip unzip = new Unzip();
         unzip.setSrc(amqStreamsZipFile);
         unzip.setDest(amqStreamsDirectory);
+        logger.info("Unpacking AMQ streams zip from {} to {}", amqStreamsZipFile.getAbsolutePath(),
+                    amqStreamsDirectory.getAbsolutePath());
         unzip.execute();
 
         return amqStreamsDirectory;
     }
 
+    private String buildHACEPImage() {
+        project.runOcCommandAsAdmin("new-build", "--binary", "--strategy=docker",
+                                    "--name", "openshift-kie-springboot");
+        final File springModuleDir = new File(OpenShiftConstants.getHaCepSourcesDir(), "springboot");
+        logger.info("Building HA-CEP Spring boot image");
+        final String buildOutput = project.runOcCommandAsAdmin("start-build", "openshift-kie-springboot",
+                                    "--from-dir=" + springModuleDir.getAbsolutePath(), "--follow");
+        logger.info(buildOutput);
+        final String imageURL = project.getOpenShiftAdmin().getImageStream("openshift-kie-springboot").getStatus().getDockerImageRepository();
```

Comment:
Name imageURL for String can be confusing as we could expect URL object behind this name.
```suggestion
        final String dockerImageRepository = project.getOpenShiftAdmin().getImageStream("openshift-kie-springboot").getStatus().getDockerImageRepository();
```

Reply:
Good point. Variable was renamed.